feature-libs/cart/base/core/facade/multi-cart.service.ts

```diff
@@ -227,7 +227,8 @@ export class MultiCartService implements MultiCartFacade {
     cartId: string,
     productCode: string,
     quantity: number,
-    pickupStore?: string
+    pickupStore?: string,
+    numberOfEntriesBeforeAdd?: number
```

Comment:
Can you help me understand why we have this new parameter of the method `addEntry()?

IMHO it should not be the responsibility of the consumer of the MultiCartService to tell how many products were there in the cart before the addition. As a consumer, I want to only tell which _new item_ I'm putting into the cart.

Reply:
Agree to your point. It's better to fetch the number of entries before add in the actual multi-cart.service implementation